app/Http/Controllers/StudentsCouncil/AnonymousQuestionController.php

```diff
@@ -37,36 +35,34 @@ public function __construct()
      * the list of questions
      * and the option to add new ones.
      */
-    public function indexSemesters()
+    public function index()
     {
         $this->authorize('administer', AnswerSheet::class);
 
-        return view('student-council.anonymous-questions.index_semesters');
+        return view('student-council.anonymous-questions.index');
     }
 
     /**
      * Returns the 'new question' page.
      */
-    public function create(Semester $semester)
+    public function create()
     {
         $this->authorize('administer', AnswerSheet::class);
 
-        if ($semester->isClosed()) {
+        if (!$this->isActive()) {
```

Comment:
I think we do need the semester here. Otherwise, we cannot reliably determine for which semester we should create the question (especially if no periodic event exists yet).

Reply:
We set the semester for the periodic event that you can use (See the store function, I'm getting the periodic Event and setting the question for that semester). We should not use the dates for the semesters as they are not reliable - see our original issue before the periodic events got introduced.

What do you want to achieve?

Reply:
We need to determine on the index page which semester is the currently active one. If only the one with an open periodic event is allowed to be added questions, the user cannot add a question to a semester whose evaluation form has not yet been opened. So they have to open it first and _then_ add the question, which is not ideal.

What if we allow every semester which is the current one or is in the future? This only affects the council, not ordinary collegists.

Reply:
I see. So instead of checking that the periodic event is active, you can check $periodicEvent->endDate()->isPast() and allow future forms to be modified.